app/javascript/spec/copy-dashboard-form/copy-dashboard-form.spec.js

```diff
@@ -67,7 +67,6 @@ describe('Copy Dashboard form', () => {
     expect(wrapper.find(MiqFormRenderer)).toHaveLength(1);
     expect(wrapper.find('input[name="name"]').instance().value).toEqual('Clint');
     expect(wrapper.find('input[name="description"]').instance().value).toEqual('good dashboard');
-    expect(wrapper.find('input[name="group_id"]').instance().value).toEqual('12');
```

Comment:
unrelated change?

Reply:
There is no longer an input. It's done via patternfly dropdown. We could technically check the span HTML or whatever its using.

Reply:
Ah, never mind.

It is the only test that makes sure group_id works, that's why I didn't like it.
But all it really does is test that a select works in ddf, and ..there's probably other tests for that :).

Reply:
Yeah